language/documentation/tutorial/README.md

```diff
@@ -41,9 +41,9 @@ Follow the prompt to install all the necessary dependencies.
 ```bash
 $ . ~/.profile
 ````
-- Install Move CLI by running this command in diem repo:
+- Install Move's CLI's by running this command in diem repo:
 ```bash
-$ cargo build -p df-cli
+$ cargo build -p df-cli -p move-analyzer
```

Comment:
It may be easier to do `cargo install` so you don't have to follow up with a copy/move `mv filename` step and can just rely on ~/.cargo/bin.

```bash
cargo install --path diem-move/df-cli
cargo install --path language/move-analyzer

Reply:
I totally agree, but see above -- tl;dr: if you don't mind, I'll send out a separate pull request to update the Move tutorial to use `cargo install` instead of `cargo build` & aliases.